### ISSUE TITLE
Lone quantifier bracket should not throw when unicode flag is not enabled

### ISSUE THREAD

**Issue by JLHwung** [jviereck/regjsparser#95]
REPL: https://runkit.com/embed/etooxrk5pmq3

Expanded Atom can be Extended Pattern Character, [spec](https://tc39.es/ecma262/#prod-annexB-ExtendedAtom)
```
ExtendedPatternCharacter ::
  SourceCharacterbut not one of ^ $ \ . * + ? ( ) [ |
```

**Comment by jviereck** [comment]
This should be fixed by #102. Can you confirm?

**Comment by JLHwung** [comment]
@jviereck Yep I can not reproduce on 0.6.4. Thanks!
